content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
We can remove all the 1.6.x boilerplate if we merge @geeknoids code fix: #6783

Reply:
Latest commit pulls in @geeknoid #6783 and removes all the 1.6.x folders